estate/models/estate_property_type.py

```diff
@@ -0,0 +1,19 @@
+from odoo import api, fields, models
+
+
+class EstatePropertyType(models.Model):
+    _name = "estate.property.type"
+    _description = "Type of estate"
+    _order = "sequence, name"
+
+    name = fields.Char(required=True)
+    property_ids = fields.One2many("estate.property", "property_type_id", string="Property")
+    sequence = fields.Integer(string='Sequence')
+    offer_ids = fields.One2many("estate.property.offer", "property_type_id")
+    offer_count = fields.Integer(compute="_compute_offer_count")
+
+    @api.depends("offer_ids")
+    def _compute_offer_count(self):
+        for type in self:
+            type.offer_count = len(type.offer_ids)
+        return True
```

Comment:
```suggestion
    def _compute_offer_count(self):
        for type in self:
            type.offer_count = len(type.offer_ids)
```
a compute method should not return anything